__tests__/pages/_document.test.js

```diff
@@ -1,9 +1,16 @@
 import '@testing-library/jest-dom'
 import MyDocument from '../../pages/_document'
+import * as utils from '../../lib/Utils.js'
 
 describe('_document', () => {
+  jest.spyOn(utils, 'generateNonce').mockReturnValue('test')
```

Comment:
Suggestion: This seems to be the more consistent way we've mocked tests in our apps
```suggestion
  jest.mock('../../lib/Utils', () => ({
    generateNonce: jest.fn().mockReturnValue('test'),
  }))
```

Reply:
jest.spyOn() is perfect on if we want to keep track of when generateNonce is being called. [Documentation](https://jestjs.io/docs/jest-object#mock-functions)

Reply:
No problem, if you think it's better to keep as is; however that benefit isn't getting used in the test. It also seems some of that call tracking is enabled on .mock now... can see what Tom's example with it was https://github.com/DTS-STN/sc-digital-centre/blob/dev/lib/api/mapBenefits.test.js#L35